2021-finalproject/Bash-file

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
Is this script write by yourself?

Reply:
OK.That was not written by myself. I had delete it.And surely I don't know how to write the scrip.I'm sorry! And I will try my best to learn it!